src/Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions/IConnectionInformation.cs

```diff
@@ -12,9 +12,10 @@ public interface IConnectionInformation
         IPEndPoint LocalEndPoint { get; }
 
         PipeFactory PipeFactory { get; }
+        IScheduler InputReaderScheduler { get; }
```

Comment:
The motivation for this is to avoid unnecessary dispatch.  Socket completion calls are already running on the thread pool, so there is no reason to dispatch again to the thread pool.

I'm happy to do this some other way though; what do you suggest?